src/modules/site-v2/base/views/auth/oauth.py

```diff
@@ -56,6 +60,29 @@ def authorized(google_bp, token):
   assert user_info.ok
   user_info = {'google': user_info.json()}
   user = create_or_update_google_user(user_info)
-
+  resp = make_response(assign_access_refresh_tokens(user.name, user.roles, session.get("login_referrer")))
+  new_resp = transfer_cart(resp, user)
   flash("Successfully logged in!", 'success')
-  return assign_access_refresh_tokens(user.name, user.roles, session.get("login_referrer"))
+  return new_resp
+
+
+def transfer_cart(resp, user):
+  # checks if a user has a local cart
+  cartID = request.cookies.get('cartID')
```

Comment:
@orzut -- one more stylistic note: in python the norm is to use lower-case var names with underscores, instead of `cartID =` consider using `cart_id =`. 

The cookie name is fine as is, and we might want to promote it to an environment variable (env/ENV/global.env) that is unique to each environment. For instance, on the env/development/global.env it could be: `MODULE_SITE_CART_COOKIE_NAME=caendr-dev-cart-id` and then use it here as:
```
MODULE_SITE_CART_COOKIE_NAME = os.getenv('MODULE_SITE_CART_COOKIE_NAME')
cart_id = request.cookies.get(MODULE_SITE_CART_COOKIE_NAME)
```